build-tests/api-documenter-test/etc/markdown/api-documenter-test.idocinterface5.md

```diff
@@ -0,0 +1,20 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
```

Comment:
@Vitalius1 FYI these new files should have been added to Git in your PR https://github.com/microsoft/web-build-tools/pull/1294

Reply:
Ohhh... I forgot to run the markdown documenter... Will pay more attention in the future. Thanks for fixing this 👍